docs/authoring/characters.md

```diff
@@ -1,250 +1,436 @@
+
+  
 
 Now for the specific emotions.
 
+  
+
 ##### `<comment>`
 
-Emote comments will display in the dropdown menu and on the emote button itself if an emote icon for it could not be found.
+  
 
-Some char.ini files might contain odd symbols; this is because the creator of Attorney Online (FanatSors) happened to be Russian and wrote many character files in Cyrillic, which Notepad doesn't read well.
+Emote **comments** will display in the dropdown menu and on the emote button itself if an emote icon for it could not be found. You can think of this as the "name" of the emote. You should make this pretty short - if it's too long, it'll probably get shortened by your theme.
 
 ##### `<preanim>`
 
-The next section defines the preanimation, or the gif played before the character actually starts speaking. If there is none, a placeholder called `normal` or `-` is typically used.
+  
+
+The next section defines the **preanimation**, or the animation played before the character actually starts speaking. If there is none, a placeholder called `-` is typically used. These can be either in the root of the character folder or stored in a subfolder (generally named "anim"). To use a preanimation file from a subfolder, you must prefix the file name with the name of the folder. For example, a preanimation of `anim/deskslam` would correspond to this structure:
+```
+char.ini
+anim/
+	deskslam.gif
+```
+This works with multiple subfolders as well - for example, `anim/young/damage`:
+```
+char.ini
+anim/
+	young/
+		damage.gif
+```
+  Consider using subfolders if your character has a lot of files.
+  
 
 ##### `<emote>` (idle and speaking animations)
 
+  
+
 Between the next #'s is the name of the actual animation when the character is idle and speaking, starting with `(a)` and `(b)` in the character folder, respectively. For instance, if your animation is named "cough", then the engine expects your idle animation to be named `(a)cough.gif` on the file system, and your talking animation to be named `(b)cough.gif`.
 
+  
+
 The engine will also scan for other supported file formats, such as `.apng` and `.webp`, as described above.
 
+  
+
 If no animations are found for an emote, the engine will fall back to static emotes. Static emotes don't use `(a)`/`(b)` prefixes - instead, an static emote named `cough` would be expected to be in the file `cough.png`.
 
+If you're the type who likes to organize, you can also place files in **subfolders**. To use subfolder mode, you simply have to prefix your emotes with a forward slash (`/`). For `(a)`/`(b)` type emotes in subfolder mode, create two folders named `(a)` and `(b)` and place your prefix-less files in the appropriate folders. For example, the emote `/angry` will search for a file named `angry.gif` in `(a)/` and `(b)/` for idle and talking respectively.
```

Comment:
```suggestion
If you're the type who likes to organize, you can also place files in subfolders, though the method is somewhat different from that of preanimations. To use subfolders, you must prefix your emotes with a forward slash (`/`). For `(a)`/`(b)` type emotes in subfolder mode, create two folders named `(a)` and `(b)` and place your prefix-less files in the appropriate folders. For example, the emote `/angry` will search for a file named `angry.gif` in `(a)/` and `(b)/` for idle and talking respectively.
```
funny how you call it "subfolder mode" even though this is just a side effect of garbo tier path concatenation

Reply:
it's not a bug, it's a feature